google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -416,14 +416,14 @@ void populateCondition(Rule.Condition condition) {
      * <a href="https://cloud.google.com/storage/docs/storage-classes">here</a>.
      */
     @Deprecated
-    public abstract Builder storageClass(String storageClass);
+    public abstract Builder storageClass(StorageClass storageClass);
 
     /**
      * Sets the bucket's storage class. This defines how blobs in the bucket are stored and
      * determines the SLA and the cost of storage. A list of supported values is available
      * <a href="https://cloud.google.com/storage/docs/storage-classes">here</a>.
      */
-    public abstract Builder setStorageClass(String storageClass);
+    public abstract Builder setStorageClass(StorageClass storageClass);
```

Comment:
I would recommend to avoid adding any new deprecated methods before GA (since we are going to remove them soon anyway)